packages/vm/src/modules/file_system_cache.rs

```diff
@@ -125,17 +129,20 @@ impl FileSystemCache {
     }
 
     /// Stores a serialized module to the file system. Returns the size of the serialized module.
-    pub fn store(&mut self, checksum: &Checksum, module: &Module) -> VmResult<()> {
+    /// The serialized module size is a good approximation (~100.06 %) of the in-memory module size.
```

Comment:
This percentage (`~100.06%`) was derived by comparing the serialised size to the size of the memory reported by `massif`. As such, is not considering `mmapp`ed memory, used in the module's storage.

Better to remove this comment, or update it to account for the size discrepancy due to mmapped storage.

We can also introduce a factor to multiply this module size, to get a better estimation using the old method.

Reply:
Okay, good point. Thank you!

I created https://github.com/CosmWasm/cosmwasm/issues/1666 for this such that we don't get blocked here.

I'll remove this comment and leave `estimate_module_size` to be updated.

Reply:
Oh, also `store` is suppsed to do that: `Returns the size of the serialized module.`. Which is a good point. The file system cache should not worry about the in-memory size estimation.